.github/workflows/main.yml

```
@@ -1,29 +1,47 @@
 name: OpenTelemetry Zipkin Tests
 
-on: [push]
+on:
+  pull_request:
+    branches:
+      - 'master'
+  push:
+    branches:
+      - 'master'
 
 jobs:
   build:
     name: Test on OTP ${{ matrix.otp_version }} and ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-
-    services:
-      zipkin:
-        image: openzipkin/zipkin-slim
-        ports:
-        - 9411:9411
-
     strategy:
       matrix:
-        otp_version: [21, 22]
-        os: [ubuntu-latest]
-
-    container:
-      image: erlang:${{ matrix.otp_version }}
-
+        otp_version: ['24.0.2', '23.3.4.2', '22.3.4.20']
```

Is it possible to use the latest generally available versions of OTP?
```suggestion
        otp_version: ['24.0.6', '23.3.4.7', '22.3.4.21']
```